astronomer/airflow/version_check/plugin.py

```diff
@@ -78,6 +90,70 @@ def start_update_thread(cls):
         AstronomerVersionCheck.ensure_singleton()
         CheckThread().start()
 
+    @classmethod
+    def get_migration_columns(cls):
+        """Return the columns to be added during migration."""
+        from .models import AstronomerAvailableVersion
+
+        return {
+            AstronomerAvailableVersion.__tablename__: [
+                Column('end_of_support', UtcDateTime(timezone=True), nullable=True),
+                Column('eos_dismissed_until', UtcDateTime(timezone=True), nullable=True),
+            ],
+        }
+
+    @classmethod
+    def migrate_db_tables_and_upsert(cls):
+        """Apply migrations to the DB tables and upsert data."""
```

Comment:
This method name and docstring are now out of date. Probably just back to `migrate_db_tables`.

Reply:
Changed it to `migrate_db_tables_and_reset_version_check`